src/main/java/com/coveros/selenified/element/check/Excludes.java

```diff
@@ -258,21 +258,21 @@ default String checkValue(String expectedValue, double waitFor, double timeTook)
      * @return String[]: all of the select values of the element. null will be returned if the element isn't present or a select
      */
     @SuppressWarnings("squid:S1168")
-    default String[] checkSelectValue(String selectValue, double waitFor, double timeTook) {
+    String[] checkSelectValue(String selectValue, double waitFor, double timeTook) {
         // record the action, and check for select
-        if (!isPresentSelect(getElement().prettyOutput() + " without a select value of <b>" + selectValue +
+        if (!isPresentSelect(this.element.prettyOutput() + " without a select value of <b>" + selectValue +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " without a select value of <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)